### DISCUSSION TITLE
Switch Statement

### DISCUSSION THREAD

**Discussion by Craafter** [SkriptLang/Skript#4880] [Suggestions] [answered]
Bye! How's it going? I'm here to propose the idea for the new version 2.3.3: the switch statement? Check numbers and strings, maybe a sample script

command /switch <text>:
    trigger:
        switch arg-1:
          case 1:
                ...
            case 2:
                ...
           default:
               ....

**Comment by AyhamAl-Ali** [comment, marked as answer]
Duplicate of #4000
